psi4/src/psi4/libtrans/integraltransform_sort_mo_tpdm.cc

```diff
@@ -77,8 +91,11 @@ void IntegralTransform::presort_mo_tpdm_restricted() {
     bucketOffset[0] = init_int_array(nirreps_);
     int **bucketRowDim = (int **)malloc(sizeof(int *));
     bucketRowDim[0] = init_int_array(nirreps_);
-    int **bucketSize = (int **)malloc(sizeof(int *));
-    bucketSize[0] = init_int_array(nirreps_);
+    //int **bucketSize = (int **)malloc(sizeof(int *));
+    //bucketSize[0] = init_int_array(nirreps_);
```

Comment:
for my part I like replace over comment and add

Reply:
I generally agree. This habit is a relic of the pre-git days of code development.  However, in this particular case there are some other 'ints' in the function that could someday need converted to size_t (but not in the immediate future) so the relics of this change may make that future fix easier.